.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: "\U0001F41B Bug Report"
+about: If you are pretty sure you are getting a bug, please create a report to help us improve this work
+title: ''
+labels: bug
+assignees: ''
+
+---
```

Comment:
can you convert this into the yaml style template that we've got for our stories please?

Reply:
sorted out here 4ecfdb1